roles/pulp_common/README.md

```diff
@@ -100,6 +100,7 @@ Role Variables
   subscription-manager/katello.
   Also accepts a single string or empty string.
   Only affects RHEL7 (RHEL8 no longer has an optional repo.)
+* pulp_disable_selinux: Set to `true` if SELinux should be disabled. Defaults to `false`.
```

Comment:
```suggestion
```
Let's hide this so we can remove it more easily later once we start installing the policy. Once we start installing the policy, it is not the job of the pulp_installer to disable SELinux, users can do that via their own playbook/roles if they wish to.

Reply:
Not sure why we're being so binary about SELinux. I think giving the user a knob to turn ad control its state is a nice feature, and makes the installer a bit more complete. What if I don't want to run SELinux in my dev environment? Just set the var, and done. I don't have to go read up on the SELinux Ansible module.

Reply:
This comment is outdated. See latest commit.